### ISSUE TITLE
Unsupported formats need handling when opening via CC Lib asset edit original

### ISSUE THREAD

**Issue by chadrolfs** [adobe-photoshop/spaces-design#2545]
Open a library containing unsupported formats (i.e. 3D)
drag asset onto canvas
double-click asset on canvas or layers panel 

Result: Opens and can edit by attempting to drag out a guide

closing results in: `Action documents.close failed: TypeError: Cannot read property 'call' of undefined`

Additional bug: clicking on title bar blinks toolbar

**Comment by chadrolfs** [comment]
https://www.dropbox.com/s/u3so3c9us9gmraw/CCLib-Open3D.mov?dl=0

**Comment by shaoshing** [comment]
The panels and guides are not completely disabled for unsupported documents. Things I found and will be fixed:

* guide creation when guide is enabled in classic
* copy/paste appearance/style buttons
* add layer effect button
* vector operation buttons
* should greyout libraries panel

**Comment by chadrolfs** [comment]
Waiting on #2562 to be fixed in order to verify this.

**Comment by shaoshing** [comment]
#2562 is replaced by #2743

**Comment by chadrolfs** [comment]
This is a case specific to 3D so let me know if you want me to open as a separate issue and close this one.

1. you can still edit/change a 3D image
https://www.dropbox.com/s/ib374gz8g6sko0y/3D-edit-pt1.mov?dl=0

2. bad things ensue after you do if you try to edit again
https://www.dropbox.com/s/2m7goi49fhx9pnm/3D-edit-pt2.mov?dl=0

**Comment by shaoshing** [comment]
@chadrolfs , 3D or other unsupported layers are living inside a PSD file, so we don't know if the PSD is supported or not until we open it and check its layer attributes, so the only option we have is to not update the changes to the graphic when you try to save a unsupported document. I've created a PR for this. Regarding this video https://www.dropbox.com/s/ib374gz8g6sko0y/3D-edit-pt1.mov?dl=0, I have difficulty on reproduce it; I did reproduce once then it seems to be disappeared. Do you have a clue of the exact steps to repdouce it?

**Comment by chadrolfs** [comment]
You need to click outside the 3D shape on the grid.  Sometime single click on the top of the 3D object then quickly back on the grid outside it and drag

**Comment by chadrolfs** [comment]
after a move/rotate or two, you can get to a point where you can't move again:
https://www.dropbox.com/s/4suxl3u0norm3jm/Move3D.mov?dl=0

**Comment by mcilroyc** [comment]
FBNC to @chadrolfs

**Comment by shaoshing** [comment]
I forgot to tell Cory that this is not complete fixed. I sill have to investigate why you can edit the 3D layer at the beginning.
